src/static/js/modules/jquery/custom-select.js

```diff
@@ -0,0 +1,81 @@
+'use strict';
+
+function init() {
+
+  $.fn.customSelect = function( userSettings ) {
+
+    function getSelectedOptionText($customSelect) {
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/bebb217d926085fc87d071fa3cb51ac548f9dc34